util/k8s/k8sApplicationService.go

```diff
@@ -52,6 +56,12 @@ type ResourceRequestBean struct {
 	K8sRequest    *application.K8sRequestBean `json:"k8sRequest"`
 }
 
+const DevtronDefaultNamespace = "devtroncd"
```

Comment:
"devtroncd" can be used from some other place (I assume this is being used somewhere)